marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/data-services/mapping/getMappingFunctions.sjs

```diff
@@ -18,4 +18,16 @@ const esMappingLib = require("/data-hub/5/builtins/steps/mapping/entity-services
 
 let mlFunctions = esMappingLib.getMarkLogicMappingFunctions();
 let xpathFunctions = esMappingLib.getXpathMappingFunctions();
-Object.assign({}, mlFunctions, xpathFunctions);
+let unOrderedMappingFunctions = Object.assign({}, mlFunctions, xpathFunctions);
+let orderedMappingFunctions = {};
```

Comment:
@srinathgit @xnikhil08 How easily can this return an array instead of an object? An array seems expected here and also addresses @ryanjdew 's concern about the order of keys not being guaranteed.

Reply:
The endpoint now returns an array.